src/xtgeo/cube/_cube_export.py

```diff
@@ -11,53 +14,68 @@
 from xtgeo._cxtgeo import XTGeoCLibError
 from xtgeo.common import XTGeoDialog, null_logger
 
+if TYPE_CHECKING:
+    from xtgeo.cube.cube1 import Cube
+
+
 logger = null_logger(__name__)
 xtg = XTGeoDialog()
 
 
-def export_segy(self, sfile, template=None, pristine=False, engine="xtgeo"):
+def export_segy(
+    cube: "Cube",
+    sfile: str,
+    template: str | None = None,
+    pristine: bool = False,
+    engine: str = "xtgeo",
+) -> None:
     """Export on SEGY using segyio library.
 
     Args:
-        self (:class:`xtgeo.cube.Cube`): The instance
+        cube (:class:`xtgeo.cube.Cube`): The instance
         sfile (str): File name to export to.
         template (str): Use an existing file a template.
         pristine (bool): Make SEGY from scrtach if True; otherwise use an
             existing SEGY file.
         engine (str): Use 'xtgeo' or (later?) 'segyio'
```

Comment:
Replacing `self` with `cube` is OK with me. Since the typing now explains datatype, you can in general remove the info in the parantheses under `Args:` e.g.

sfile ~~(str)~~: File name to export 

==>

sfile: File name to export

Note also, FYI, that docstrings in modules preceeded by an underscore, e.g. `_ cube_export.py` will not be a part of the public API docs